File: cpp/produk.cpp
#include <bits/stdc++.h>
using namespace std;

class produk // Class produk
{
private: // Atribut
    string id;
    string nama;
    int harga;

public:
    // Constructor
    produk() {}

    // Getter
    string getId() const { return this->id; }
    string getNama() const { return this->nama; }
    int getHarga() const { return this->harga; }

    // Setter
    void setId(string id) { this->id = id; }
    void setNama(string nama) { this->nama = nama; }
    void setHarga(int harga) { this->harga = harga; }

    // Destructor
    ~produk() {}
};

File: java/Produk.java
public class Produk 
{
    private String id;
    private String nama;
    private int harga;

    // Constructor
    public Produk() 
    {
        this.id = "";
        this.nama = "";
        this.harga = 0;
    }

    // Getter
    public String getId() { return this.id; }
    public String getNama() { return this.nama; }
    public int getHarga() { return this.harga; }

    // Setter
    public void setId(String id) { this.id = id; }
    public void setNama(String nama) { this.nama = nama; }
    public void setHarga(int harga) { this.harga = harga; }
}